### PR TITLE
I have done nothing but code bread for three hours

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request

teleporting bread now mutates it
teleportations 1-4 add mutagen
teleportation 5 turns it into a monster bread
the monster bread gets healed and gains power when teleported, with a random cap.

## Why It's Good For The Game
 why not

## Changelog
:cl:zeskorion
add: teleporting bread is now dangerous
code: adds teleport_act to atoms
imageadd: killer bread sprites
/:cl:

### CODE REVIEW

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Super dirty branch, rebase to get rid of the extra commits please.

**Comment by zeskorion** [comment]
aw fuck i just made things worse. sorry, still gettin the hang of this thing...

**Comment by zeskorion** [comment]
aaand there we go

**Comment by Crossedfall** [comment]
eeh, not quite. The `beestation.dme` is still all borked and adding 6,000 line changes to your relatively small PR.

**Comment by ike709** [comment]
Pretty sure there is already a chance for this to happen on TG.

**Comment by ike709** [comment]
There was an attempt: https://github.com/tgstation/tgstation/pull/41503

**Comment by ike709** [comment]
Change the line endings for the dme in VSCode to fix the issues.

**Comment by zeskorion** [comment]
I'll fix it when i get home then

File: beestation.dme [hunk 1]
```diff
@@ -264,8 +264,8 @@
 #include "code\controllers\subsystem\pathfinder.dm"
 #include "code\controllers\subsystem\persistence.dm"
 #include "code\controllers\subsystem\ping.dm"
-#include "code\controllers\subsystem\profiler.dm"
 #include "code\controllers\subsystem\radiation.dm"
+#include "code\controllers\subsystem\profiler.dm"
```
Comment (ike709) [inline beestation.dme]: This is out of order and will cause conflicts down the road. The DME should be alphabetical.

File: beestation.dme [hunk 2]
```diff
@@ -2924,7 +2925,8 @@
 #include "code\modules\surgery\advanced\bioware\experimental_dissection.dm"
 #include "code\modules\surgery\advanced\bioware\ligament_hook.dm"
 #include "code\modules\surgery\advanced\bioware\ligament_reinforcement.dm"
-#include "code\modules\surgery\advanced\bioware\muscled_veins.dm"
+#include "code\modules\surgery\advanced\bioware\muscled_veins.dm"
+
```
Comment (ike709) [inline beestation.dme]: Whitespace...?

**Comment by ike709** [comment]
Only reviewed the DME to see if it was fixed.

**Comment by zeskorion** [comment]
alright, after 5 fuckin years here we go. fuck that file.

File: code/modules/mob/living/simple_animal/hostile/bread.dm [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	icon_state = "mutantbread"
+	icon_living = "mutantbread"
+	icon_dead = "bread"
+	gender = NEUTER
+	speak_chance = 0
+	turns_per_move = 5
+	maxHealth = 30
+	health = 30
+	see_in_dark = 3
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/breadslice = 2)
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/hostile/bread.dm]: inb4 people use this to dupe breadslices

**Review by ike709 — APPROVED** [review]

**Review by Crossedfall — APPROVED** [review]

File: code/game/atoms.dm [hunk 4]
```diff
@@ -666,6 +666,14 @@
 	SEND_SIGNAL(src, COMSIG_ATOM_RCD_ACT, user, the_rcd, passed_mode)
 	return FALSE
 
+/**
+  * Respond to our atom being teleported
+  *
+  * Default behaviour is to send COMSIG_ATOM_TELEPORT_ACT and return
+  */
+/atom/proc/teleport_act()
+	SEND_SIGNAL(src,COMSIG_ATOM_TELEPORT_ACT)
```
Comment (qwertyquerty) [inline code/game/atoms.dm:675]: Why have a signal for it if you never use it?

Comment (zeskorion) [reply code/game/atoms.dm:675]: for future use, mostly. also, just about every other _act proc has that line, and most dont seem to use it

**Review by CthulhuOnIce — APPROVED** [review]
Mmm myes more references

**Comment by CthulhuOnIce** [comment]
> reviewing and approving a 2 month old pr because someone posted it in chat today

😔

**Pull request merged**